digitaltwin_client/samples/digitaltwin_sample_environmental_sensor/digitaltwin_sample_environmental_sensor.c

```diff
@@ -1,11 +1,10 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-// Implements a sample DigitalTwin interface that integrates with an environmental sensor (for reporting 
-// temperature and humidity over time).  It also has basic commands and properties it can process,
-// such as setting brightness of a light and blinking LEDs.  Because this sample is designed
-// to be run anywhere, all of the sameple data and command processing is expressed simply with 
-// random numbers and LogInfo() calls.
+// Implements a sample DigitalTwin interface as defined by "dtmi:my_company:com:EnvironmentalSensor;1"
```

Comment:
does the user know where to find this `dtmi:my_company:com:EnvironmentalSensor;1`?
at least the json document in this directory has a value named `dtmi:com:example:EnvironmentalSensor;1`

Reply:
Yeah - that mismatch is no good.  I'm updating .c to reflect the JSON in dir.  Ultimately this JSON will be in a central spot all SDKs can point to (#1529 tracks) so the long-term documenting of this only will be solved once we get a spot to point at but not until then.